src/main/scala/views/language_selection.scala.html

```diff
@@ -14,19 +14,19 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.urls.Link
+@import views.Link
 
 @(langMap: Map[String, Lang], langToCall: String => Call, customClass: Option[String] = None, appName: Option[String] = None)(implicit messages: Messages)
 
 <p class="@if(customClass.isDefined) {@customClass.get}">
 
     @langMap.map { case (key: String, value: Lang) =>
         @if(messages.lang.code != value.code) {
-            @Link.toInternalPage(
+            @Link(
```

Comment:
to be honest I'd just write an inline html link here @ellamdav I don't think we'll want to reuse this or change it in the future, this html fragment is an old bit of functionality only (maybe) still used by assets-frontend services

Reply:
Inlined the code, added a test for when the `data-journey-click` attribute should not be included, otherwise covered by existing tests.  Bumped major version to reflect breaking change.